app/assets/sass/nhs_fonts.scss

```diff
@@ -1,8 +1,20 @@
-
-
 @font-face {
   font-family: 'nta';
   src: url(/public/fonts/NeueFrutigerComRegular.woff);
   font-weight: normal;
   font-style: normal;
-}
\ No newline at end of file
+}
+
+@font-face {
+  font-family: 'Frutiger Regular';
```

Comment:
It's possible to use the same font-family name for multiple weights, and the browser should then use the correct font file when rendering bold text.

Is there a reason to use this approach with multiple font-family names instead?